roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -46,18 +46,25 @@ def run(self, tmp=None, task_vars=None):
 
         result["checks"] = check_results = {}
 
+        user_disabled_checks = [
+            check.strip()
+            for check in task_vars.get("openshift_disable_check", "").split(",")
```

Comment:
Not really. Presumably there will eventually be docs for this.

The checks that did fail are mentioned in the output. We could make that more obvious with a summary line so that after a failure the user can review and decide if they don't care about the failures or want to reconfigure thresholds etc.